docs/quick_start.md

```diff
@@ -16,38 +16,88 @@ limitations under the License.
 
 # Quick Start
 
-The steps below will guide you through using model analyzer to analyze a simple PyTorch model.
+The steps below will guide you through using model analyzer to analyze a simple PyTorch model: add_sub.
```

Comment:
We need these steps protected via a test.

Reply:
Follow-on is fine as long as you have manually run the steps. I can double check once you think they are ready.

Reply:
Yes, I have manually run the steps.

Reply:
ok is there a follow-on story filed?

Reply:
TMA-725